src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days < 3) {
+    return total;
+  }
+
+  if (days < 7) {
+    return total - 20;
```

Comment:
You should create variables for discount

Reply:
> Great job!

Thank you very much, for helping me see the flaws in my code!👍